documentation/builders/installation.md

```diff
@@ -77,6 +77,39 @@ You will need a terminal, like PuTTY for Windows or the Terminal app for Mac to
 
 </details>
 
+### Pre-install preparation / workarounds
+
+#### Workaround for network related features on Bookworm
+<details>
+With Bookworm the network settings have changed. Now "NetworkManager" is used instead of "dhcpcd".
+This breaks breaks network related features like "Static IP", "Wifi Setup" and "Autohotspot".
+Before running the installation, the network config has to be changed via raspi-config, to use the "old" dhcpcd network settings.
```

Comment:
If we refer to use Bullseye, this is not a problem anymore.
Meanwhile, can't we just automate this instead of describing it here, potentially loosing builders along the way? We could silently change the raspi-config in the installation, most builders won't care. It's a save convention to define.

Reply:
We could keep this docs for reference on Bookworm, but it'll be less of a pain for most builders.

Reply:
Our current installation instruction dont mention a specific version (the default will be select be RPI Imager), but i  think more and more will use the newer Bookworm, and running into this. 

A quickfix in the installation is not possible, as a change of the setting will instantly reset the network and the user will lose access to the device, if it has a wifi connection.

I adress the problem, with https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/2184 to have both ways working. 
So this is just a temporary documentation apart from the Issue and notes in the Release till the implementation is finished ( in case it will not be ready for 3.5.0 ).